staging/src/k8s.io/kubectl/pkg/cmd/auth/whoami.go

```diff
@@ -22,14 +22,17 @@ import (
 	"io"
```

Comment:
@soltysh - should this be wired into NewCmdAuth as part of promotion to beta? I'm fuzzy on how alpha commands are intended to progress in kubectl

Reply:
Jordan and I spoke on slack, but for visibility, the guidelines for promoting commands from alpha to beta are similar to any other API. The only difference is mechanics:
- for new subcommands you move out of `kubectl alpha` to appropriate place
- for new features added to pre-existing command (new flags, functionality) the functionality is on by default, ie. no env var required.

Reply:
Great, thank you for the explanation.